proprietary/Components/src/denhaag/publication-date.tokens.json

```diff
@@ -0,0 +1,14 @@
+{
+  "denhaag": {
+    "publication-date": {
+      "border": { "value": "1px solid #D2D2D2" },
+      "color": { "value": "{denhaag.color.grey.4}" },
+      "font-family": { "value": "{denhaag.typography.sans-serif.font-family}" },
+      "font-weight": { "value": "400" },
+      "font-size": { "value": "1.125rem" },
```

Comment:
I noticed this font size is not defined in the typography tokens. We only have `1rem` and `1.125rem`. Are our typography tokens outdated? Or is this just an exception.

Reply:
@bddjong this is an exception that follows the UX documentation in figma